pkg/kubelet/volumes.go

```diff
@@ -34,7 +35,7 @@ import (
 	"k8s.io/kubernetes/pkg/volume"
 )
 
-var errUnsupportedVolumeType = fmt.Errorf("unsupported volume type")
+var errUnsupportedVolumeType = fmt.Errorf("failed volume mount, check previous pod describe events or logs for more details")
```

Comment:
@screeley44 @pmorie - I took a look at the usages of this error.  It looks like some of the usages (`NewWrapperBuilder`, `NewWrapperCleaner`) will actually hide the underlying error from the `New*` call and some don't (`mountExternalVolumes`).  Should we go ahead and standardize that too?  

I'll defer to folks who are in this code more than me.